### ISSUE TITLE
Wrong issue-URL in error message when requesting bugreport 

### ISSUE THREAD

**Issue by htgoebel** [wummel/patool#28]
The URL is this error message is wrong:
```
You have found an internal error in patool. Please write a bug report
at http://wummel.github.io/patool/issues and include at least the information below:
```

Either add a redirect page to wummel.github.io (github is able to do this somehow) or correct that URL.

**Comment by wummel** [comment]
We believe that the issue you reported is fixed in the source repository of patool which can be found under:
https://github.com/wummel/patool

Changelog entry:
* Fix support URL
  Closes: GH bug #28

Thank you for reporting the issue. It is now marked as fixed. If you believe that the issue is not fixed appropriately just add a comment to this issue.

**Comment by wummel** [comment]
A new version 1.9 of patool has been released on 07.12.2015.
Therefore this bug will be closed. If you think this issue is not solved, please open a new issue.
